content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -53,24 +53,34 @@ creating a patch to fix it in the upstream project.
 
 ## Cloning the Kubernetes repository
 
-If you don't already have the kubernetes/kubernetes repository, get it now:
+If you don't already have the kubernetes/kubernetes repository, get it now, and if you already have a fork of `kubernetes` then make sure your `kubernetes` fork is up-to-date with the `kubernetes/kubernetes` remote on
+GitHub (`master` branch), and clone your `kubernetes` fork.
 
 ```shell
 mkdir $GOPATH/src
 cd $GOPATH/src
```

Comment:
I would propose to keep the directory structure aligned to the convention mentioned in the [GitHub workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)

The GitHub workflow proposes the working directory to be
`export working_dir="$(go env GOPATH)/src/k8s.io"`

So we should align with it as follows:
```
mkdir -p $GOPATH/src/k8s.io
cd $GOPATH/src/k8s.io
```